pkgs/development/interpreters/wasmer/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, rustPlatform, fetchFromGitHub, cmake, llvmPackages, openssl, pkgconfig }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wasmer";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "wasmerio";
+    repo = pname;
+    rev = version;
+    sha256 = "045pldbgfbcv8071qs856sh9fpyzpyr5ybnsxswkyy2xsczi2xa8";
+    fetchSubmodules = true;
+  };
+
+  cargoSha256 = "1gydvj9f62qf1cfwpjrjcg3cwvh0byj91wnmid2flj2if5mnij9d";
+
+  nativeBuildInputs = [ cmake llvmPackages.libclang openssl pkgconfig ];
```

Comment:
why is openssl needed? for the command line tool?

Reply:
I think it's not needed, after some testing. Removed.